kitchen-terraform.gemspec

```diff
@@ -30,7 +30,7 @@ require "rubygems"
   specification.add_development_dependency "middleman-livereload", "~> 3.4"
   specification.add_development_dependency "middleman-syntax", "~> 3.0"
   specification.add_development_dependency "middleman", "~> 4.2"
-  specification.add_development_dependency "mini_racer", "~> 0.2.0"
+  specification.add_development_dependency "mini_racer", "~> 0.2"
```

Comment:
The original pinning resulted in mini_racer 0.2.14 as shown in gems.locked
https://github.com/newcontext-oss/kitchen-terraform/blob/5463a02bd1918b189018dc99595c6bc440651e26/gems.locked#L463

When I re-ran bin/update-gems-locked.sh (bundle update -all) to solve another Dependabot vuln, there was an error on mini_racer 0.2.15 which conflicted with libv8. There's a little discussion on mini_racer repo including https://github.com/rubyjs/mini_racer/issues/168 

Pinning to mini_racer 0.2.14 still results in failed build unless we'd try to pin libv8 too but I did not explore that. Upon further testing now, it seems build is fixed in 0.3.0 so perhaps instead of `"~> 0.2"`, I should pin to `"~> 0.3.0"`

Reply:
0.3.0 seems good to me!